extensions.json

```
@@ -644,6 +644,11 @@
       "version": "5.6.1",
       "checkout": "v5.6.1"
     },
+    {
+      "id": "rokoroku.vscode-theme-darcula",
+      "repository": "https://github.com/rokoroku/vscode-theme-darcula",
+      "version": "1.2.1"
```

Also, this project has dedicated release tags, so I think it's better to publish version `1.2.1` from the `v1.2.1` tag instead of from the repository's default branch.

```suggestion
      "version": "1.2.1",
      "checkout": "v1.2.1"
```